src/org/zaproxy/zap/spider/parser/SpiderHtmlFormParser.java

```diff
@@ -19,67 +19,56 @@
 
 import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
-import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.Date;
+import java.util.HashMap;
 import java.util.Iterator;
 import java.util.LinkedList;
 import java.util.List;
-
-import net.htmlparser.jericho.Element;
-import net.htmlparser.jericho.FormControl;
-import net.htmlparser.jericho.FormControlType;
-import net.htmlparser.jericho.FormField;
-import net.htmlparser.jericho.FormFields;
-import net.htmlparser.jericho.HTMLElementName;
-import net.htmlparser.jericho.Source;
```

Comment:
Do these really need to be moved?

It is cleaner to have them grouped nicely, however that's a formatting change and should be included as a separate PR.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting